namingserver/src/main/java/org/apache/seata/namingserver/controller/NamingController.java

```diff
@@ -96,10 +100,8 @@ public MetaResponse discovery(@RequestParam String vGroup, @RequestParam String
     }
 
     @PostMapping("/addGroup")
-    public Result<String> addGroup(@RequestParam String namespace,
-                                      @RequestParam String clusterName,
-                                      @RequestParam String unitName,
-                                      @RequestParam String vGroup) {
+    public Result<String> addGroup(@RequestParam String namespace, @RequestParam String clusterName, String unitName,
+        @RequestParam String vGroup) {
```

Comment:
```suggestion
    public Result<String> addGroup(@RequestParam String namespace, @RequestParam String clusterName, @RequestParam String unitName,
        @RequestParam String vGroup) {
```

If `unitName` is not passed as a parameter, you can ignore this comment!

Reply:
The unitname is only used for the group in Seata's Raft mode. It is not required for other storage modes, so it is not necessary to provide it.

Reply:
Thank you for the detailed explanation! :)
I think it’s okay to mark it as resolved.